app/src/main/AndroidManifest.xml

```diff
@@ -2,20 +2,47 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="edu.uc.reedws.musiclink">
 
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+
+    <uses-feature android:name="android.hardware.type.watch" />
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+
+
+<!--        <uses-library-->
```

Comment:
Whats this commented section for?

Reply:
I'm not sure what that comment section was for. It might have been auto-generated when I was experimenting with things. It has been deleted.